README.md

```diff
@@ -42,7 +42,10 @@ This project's goals are, in order:
 
 ## Installation
 
-Compile `tfautomv` from source:
```

Comment:
For macOs it is not enough. Since we are not registered developers, you also need to run something like `xattr -dr com.apple.quarantine $(which tfautomv)`